src/components/MissingSkuModal.vue

```diff
@@ -41,8 +41,12 @@
         </ion-list>
 
         <ion-list v-if="segmentSelected === 'completed'">
-          <ion-item v-for="item in getCompletedItems()" :key="item.shopifyProductSKU">
+          <ion-item v-for="item in getCompletedItems()" :key="item.shopifyProductSKU" lines="full">
```

Comment:
why have we added lines="full"?

Reply:
Looked better with lines="full" 
as compared to this

![Screenshot from 2023-03-06 12-23-46](https://user-images.githubusercontent.com/58461801/223039574-6be260cf-b933-4fe4-a707-169956398d18.png)